run-integration-tests.py

```diff
@@ -237,6 +247,22 @@ def __exit__(self, tpe, value, traceback):
         default=False,
         action="store_true",
         help="Use testpypi for testing pip installation")
+    parser.add_argument(
+        "--packages",
+        required=False,
+        default=None,
+        help="Additional packages required for some tests")
+    parser.add_argument(
+        "--jars",
+        required=False,
+        default=None,
+        help="Additional jars required for some tests")
+    parser.add_argument(
+        "--conf",
+        required=False,
+        default=None,
+        nargs="+",
```

Comment:
`+` ---> Do we require that at least one conf argument be provided?

Reply:
`+` means if `--conf` is present then at least one argument for `--conf` need to be provided.
So `--conf` alone is not accepted, you need to provide `--conf a=b` or `--conf a=b c=d`.

Reply:
`+` means if `--conf` is present then at least one argument for `--conf` need to be provided.
So `--conf` alone is not accepted, you need to provide `--conf a=b` or `--conf a=b c=d`.

Just checked it and it works as expected (`--conf` is optional)